app/HutchManager/Dto/ROCratesQuery.cs

```diff
@@ -0,0 +1,69 @@
+using System.Text.Json.Serialization;
+using Newtonsoft.Json;
+
+
+namespace HutchManager.Dto;
+
+public class ROCratesQuery
+{
+  [JsonProperty(PropertyName="@context")]
+  public string Context { get; set; } = string.Empty;
```

Comment:
```suggestion
  public string Context { get; set; } = "https://w3id.org/ro/crate/1.1/context";
```
This is what context will be according to the RO crates docs